lxc/info.go

```diff
@@ -587,29 +586,82 @@ func (c *cmdInfo) instanceInfo(d lxd.InstanceServer, remote config.Remote, name
 	for _, snap := range snaps {
 		if firstSnapshot {
 			fmt.Println(i18n.G("Snapshots:"))
+			columnNames := []string {"Name", i18n.G("Taken at"), i18n.G("Expires at"), "Stateful"}
```

Comment:
Shouldn't `Name` and `Stateful` be wrapped in `i18n.G()` as well?

Reply:
@monstermunchkin 
The reason I didn't wrap `Name` and `Stateful` in `i18n.G()` is because they are not wrapped in the current existing code while `Taken at` and `Expires at` are..

I will put everything in `i18n.G()`.